src/frontend/app/store/actions/application.actions.ts

```diff
@@ -53,7 +52,7 @@ export const DELETE_INSTANCE = '[Application Instance] Delete';
 export const DELETE_INSTANCE_SUCCESS = '[Application Instance] Delete success';
 export const DELETE_INSTANCE_FAILED = '[Application Instance] Delete failed';
 
-const applicationEntitySchema = entityFactory(applicationSchemaKey);
+export const applicationEntitySchema = entityFactory(applicationSchemaKey);
```

Comment:
Proper nit picky, we should try to ensure we never export a schema to avoid miss-use.